pages/button-dropdown/item-element.permutations.page.tsx

```diff
@@ -141,9 +150,7 @@ export default function () {
           permutations={permutations}
           render={permutation => (
             <Box margin={{ bottom: 'xxl' }} className={styles['dropdown-permutation']}>
-              <Dropdown trigger={null} open={true}>
```

Comment:
the usage of `Dropdown` here was part of the cause of recent flakiness in these screenshot tests, and doesn't seem necessary if we're just interested in testing the actual items